Makefile

```diff
@@ -285,3 +289,11 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
+install-binaries: _output/local/bin
```

Comment:
I like it, would it better to name it just as `install`?
Also, `make install WHAT=keadm`  and so on to keep sync with `make all` could be better.